pkg/coreapi/gql.schema.graphql

```diff
@@ -170,6 +171,73 @@ type FunctionEvent {
   createdAt: Time
 }
 
+type FunctionConfiguration {
+  cancellations: [CancellationConfiguration!]
```

Comment:
Nit:

```suggestion
  cancellations: [CancellationConfiguration!]!
```

That'd simplify the types ever so slightly. So no cancellations would be `[]` instead of `null`